prod-cockpit-rcar.yaml

```diff
@@ -56,8 +56,7 @@ common_data:
     - [SOC_FAMILY, "%{SOC_FAMILY}"]
 
     # Add systemd configuration
-    - [DISTRO_FEATURES_append, " systemd"]
```

Comment:
Please do not remove this append. recipe for gles-user-module is looking for this.

Reply:
Sorry, my previous comment is not valid. "systemd" is appended in `init-manager-systemd.inc`